### ISSUE TITLE
Compiled Version Anywhere?

### ISSUE THREAD

**Issue by SoftCircuits** [EWSoftware/SHFB#712]
Is there a compiled version of the software anywhere? Or is the plan that we need to build it if we want to use it?

It's a little overwhelming to get started with seemingly endless folders of examples and documentation and not an obvious path to get started. I already had this running once, and have a project file created. I'd like to update the help file.

**Comment by EWSoftware** [comment]
The latest release with guided installation can be downloaded from the [Releases page](https://github.com/EWSoftware/SHFB/releases).

See the [Getting Started topics](http://ewsoftware.github.io/SHFB/html/b772e00e-1705-4062-adb6-774826ce6700.htm) for help on using it.

**Comment by SoftCircuits** [comment]
Thanks. I got a little further. I tried opening that existing project but got an error.

> Validating and copying documentation source information
Source: D:\Users\Jonathan\source\repos\Toxic\Toxic.Desktop\Toxic.Desktop.csproj
    Found project 'D:\Users\Jonathan\source\repos\Toxic\Toxic.Desktop\Toxic.Desktop.csproj'
    Last step completed in 00:00:00.0410

> SHFB: Error BE0065: BUILD FAILED: Incompatible Visual Studio project file format.  See the error code help topic for more information.
This project may be for a newer version of MSBuild and cannot be loaded.  Error message:
   at SandcastleBuilder.Utils.MSBuild.MSBuildProject..ctor(String projectFile)
   at SandcastleBuilder.Utils.BuildEngine.BuildProcess.ValidateDocumentationSources()
   at SandcastleBuilder.Utils.BuildEngine.BuildProcess.Build()

> The default XML namespace of the project must be the MSBuild XML namespace. If the project is authored in the MSBuild 2003 format, please add xmlns="http://schemas.microsoft.com/developer/msbuild/2003" to the <Project> element. If the project has been authored in the old 1.0 or 1.2 format, please convert it to MSBuild 2003 format.  D:\Users\Jonathan\source\repos\Toxic\Toxic.Desktop\Toxic.Desktop.csproj
   at Microsoft.Build.Shared.ProjectErrorUtilities.ThrowInvalidProject(String errorSubCategoryResourceName, IElementLocation elementLocation, String resourceName, Object[] args)
   at Microsoft.Build.Construction.ProjectParser.Parse()
   at Microsoft.Build.Construction.ProjectParser.Parse(XmlDocumentWithLocation document, ProjectRootElement projectRootElement)
   at Microsoft.Build.Construction.ProjectRootElement..ctor(String path, ProjectRootElementCache projectRootElementCache, BuildEventContext buildEventContext)
   at Microsoft.Build.Construction.ProjectRootElement.CreateProjectFromPath(String projectFile, IDictionary`2 globalProperties, String toolsVersion, ILoggingService loggingService, ProjectRootElementCache projectRootElementCache, BuildEventContext buildEventContext)
   at Microsoft.Build.Construction.ProjectRootElement.<>c__DisplayClass170_0.<OpenProjectOrSolution>b__0(String path, ProjectRootElementCache cache)
   at Microsoft.Build.Evaluation.ProjectRootElementCache.Get(String projectFile, OpenProjectRootElement openProjectRootElement, Boolean isExplicitlyLoaded)
   at Microsoft.Build.Evaluation.Project..ctor(String projectFile, IDictionary`2 globalProperties, String toolsVersion, String subToolsetVersion, ProjectCollection projectCollection, ProjectLoadSettings loadSettings)
   at SandcastleBuilder.Utils.MSBuild.MSBuildProject..ctor(String projectFile)
